packages/contracts-bedrock/test/setup/Upgrade.s.sol

```diff
@@ -0,0 +1,193 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+// Testing
+import { VmSafe } from "forge-std/Vm.sol";
+import { console2 as console } from "forge-std/console2.sol";
+import { stdJson } from "forge-std/StdJson.sol";
+
+// Scripts
+import { Deployer } from "scripts/deploy/Deployer.sol";
+import { DeploySuperchainInput, DeploySuperchain, DeploySuperchainOutput } from "scripts/deploy/DeploySuperchain.s.sol";
+
+// Libraries
+import { Types } from "scripts/libraries/Types.sol";
+import { Process } from "scripts/libraries/Process.sol";
+
+/// @title Upgrade
+contract Upgrade is Deployer {
+    using stdJson for string;
+
+    string internal _stateJson;
+    string internal _stdVersionsToml;
```

Comment:
nit, can make these `private` for added safety, I don't think they should be directly accessed by other contracts